pkg/netlink/netlink.go

```diff
@@ -47,6 +47,7 @@ type Basic interface {
 	RouteGet(destination net.IP) ([]netlink.Route, error)
 	RouteList(link netlink.Link, family int) ([]netlink.Route, error)
 	FlushRouteTable(tableID int) error
+	NewTableRule(tableID int) *netlink.Rule
```

Comment:
This function doesn't need to be part of the interface, ie it can be just package scoped.